.changeset/nervous-pets-sleep.md

```diff
@@ -0,0 +1,116 @@
+---
+'@primer/react': major
+---
+
+![image](https://user-images.githubusercontent.com/2313998/155632630-60822d7b-6053-480d-925d-cea4d3b04919.png)
```

Comment:
If we go with these banners in the change logs*, can we change this to a more descriptive caption here please? 

Also, wondering what you think about switching the labels and group headings to be more product-oriented in the banner, like they are in code snippets? 

* Not sure if we are doing banners, as other PRs don't include them. We'd need to backfill them.